dualscreeninfo/android/src/main/kotlin/com/microsoft/reactnativedualscreen/dualscreen/DualScreenInfo.kt

```diff
@@ -81,27 +105,41 @@ class DualScreenInfo constructor(context: ReactApplicationContext) : ReactContex
 		return false
 	}
 
+	private fun rotationToString(rotation : Int) : String {
+		if (rotation == Surface.ROTATION_0) return "rotation0";
+		if (rotation == Surface.ROTATION_90) return "rotation90";
+		if (rotation == Surface.ROTATION_180) return "rotation180";
+		if (rotation == Surface.ROTATION_270) return "rotation270";
+
+		return "rotation0";
```

Comment:
A sensible default though I am not sure whether it has much of an effect given that almost all angles will default to "rotation0"